src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
+      out.append(msg.timestamp() + "\t"
+                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]: ");
       return out;
    }
 
 
    // helper for normal
    std::string normalToString(const LogMessage& msg) {
       auto out = LogDetailsToString(msg);
-      out.append('"' + msg.message() + '"');
+      out.append(msg.message() + '\n');
       return out;
    }
 
    // helper for fatal signal
    std::string  fatalSignalToString(const LogMessage& msg) {
       std::string out; // clear any previous text and formatting
-      out.append("\n" + msg.timestamp()
+      out.append(msg.timestamp()
                  + "\n\n***** FATAL SIGNAL RECEIVED ******* \n"
-                 + '"' + msg.message() + '"');
```

Comment:
I get used to Log4j style. If you don't like mine, it is fine.  

1. I don't agree to add new line in beginning of the every message. The new line should be added in the very end of message. It looks very weird to me, when I use sample color cout sink and also my program use std::cout at the same time. My text from std::cout follows g3log message because no new line after g3log message.

2. It is unnecessary to quote the log message.